### PR TITLE
feat: add doc update time for every time

### PR DESCRIPTION
**Pull request by yangwenmai**
## Snapshot
image

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -131,7 +136,7 @@ const indexTemplate = `
       		</select>
 		</label>
     </div>
-    <div class="col-6"></div>
+    <div class="col-6 custom-top-update">Last Updated At: {{.UpdatedAt}}</div>
```
Comment (changkun) [inline doc.go:139]:
```suggestion
    <div class="col-6 custom-top-update">Last Updated: {{.UpdatedAt}}</div>
```

File: doc.go [hunk 2]
```diff
@@ -91,6 +93,8 @@ func parseTemplate(lang, target string, b bytes.Buffer) {
 
 	tmpl := template.Must(template.New(target).Parse(indexTemplate))
 
+    now := time.Now().UTC()
+    updatedAt := now.Format("2006-01-02")
```
Comment (changkun) [inline doc.go:97]:
```suggestion
    updatedAt := time.Now().UTC().Format("2006.01.02")
```

File: doc.go (hunk 1 above)
Comment (changkun) [reply doc.go:139]: I don't understand why such date is highlighted on top of the navigation bar. It's awkward IMHO. Is it important? Can we just put it at the bottom of the page? See https://golang.design/go-questions/errata/ as an example.

Comment (yangwenmai) [reply doc.go:139]: yes, we should not highlighted on top of navigation bar, and put it at the bottom of the page.

File: doc.go [hunk 3]
```diff
@@ -91,6 +93,7 @@ func parseTemplate(lang, target string, b bytes.Buffer) {
 
 	tmpl := template.Must(template.New(target).Parse(indexTemplate))
 
+    updatedAt := time.Now().UTC().Format("2006.01.02")
```
Comment (changkun) [inline doc.go:96]:
```suggestion
```

File: doc.go [hunk 4]
```diff
@@ -100,6 +103,7 @@ func parseTemplate(lang, target string, b bytes.Buffer) {
 		Selected:   selected,
 		Navigation: template.HTML(toc),
 		Content:    template.HTML(dom),
+		UpdatedAt:  updatedAt,
```
Comment (changkun) [inline doc.go:106]:
```suggestion
		UpdatedAt:  time.Now().UTC().Format("2006.01.02"),
```

**Review by changkun — APPROVED** [review]

**Pull request merged**